bin/upgrading/proplists.dat

```diff
@@ -221,6 +221,14 @@ userproplist.disable_comm_promo:
   indexed: 0
   multihomed: 0
   prettyname: Administratively disable community promo features for a user
+  
+userproplist.displaydate_check:
+  cldversion: 4
+  datatype: bool
+  des: Status of the update_displaydate checkbox, which sets which posting time to use.
```

Comment:
Since this is a boolean, I think that "which sets which posting time to use" is a little vague. How about something like "Automatically updates the posting time in the (new) entry page"?